content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/login.md

```diff
@@ -0,0 +1,29 @@
+---
+title: influxctl auth login
+description: >
+  The `influxctl auth login` command logins with InfluxData Auth0 or InfluxDB
+  Clustered identity provider
+menu:
+  influxdb_clustered:
+    parent: influxctl auth
+weight: 301
+---
+
+The `influxctl auth login` command logins with InfluxData Auth0 or InfluxDB
+Clustered identity provider
```

Comment:
> Clustered

Does this also apply to Cloud Ded? If so, we can use the `{{< product-name omit="Clustered" >}}` short tag. If not, I can remove it here and keep it in `/clustered...`.